src/jit/compiler.h

```diff
@@ -1771,7 +1773,11 @@ class Compiler
     // a PSPSym for functions with any EH.
     bool ehNeedsPSPSym() const
     {
+#ifdef _TARGET_X86_
```

Comment:
It depends on the intricate implementation details. The current Win x86 exception handling or CoreRT exception handling on all platforms do not need PSPSym, so it is certainly possible to get away without having it ... I do not know whether it is a good idea for Linux x86.